core/package.json

```diff
@@ -91,6 +91,7 @@
 		"openapi3-ts": "^4.1.2",
 		"pg": "^8.11.3",
 		"prosemirror-markdown": "^1.12.0",
```

Comment:
why did we already have `prosemirror-markdown` ??

Reply:
hmm looks like for the `pushToV6` action! 

https://github.com/pubpub/platform/blob/1f3217c7319d29356a160726076fc1a1e6a87778/core/actions/pushToV6/run.ts#L136-L138

Reply:
yea i used it to convert markdown to a prosemirror doc before creating the pub in v6

Reply:
ahh that makes sense